app/javascript/app/components/ndcs/ndcs-explore-table/ndcs-explore-table-selectors.js

```diff
@@ -42,6 +49,50 @@ export const getDefaultColumns = createSelector(
   }
 );
 
+export const tableGetSelectedData = createSelector(
+  [getIndicatorsParsed, getCountries],
+  (indicators, countries) => {
+    if (!indicators || !indicators.length || !indicators[0].locations) {
+      return [];
+    }
+    const refIndicator =
+      indicators.find(i => i.value === 'submission') || indicators[0];
+
+    return Object.keys(refIndicator.locations).map(iso => {
+      if (refIndicator.locations[iso].value === 'No Document Submitted') {
+        return false;
+      }
+      const countryData =
+        countries.find(country => country.iso_code3 === iso) || {};
+      const row = {
+        country: countryData.wri_standard_name || iso,
+        iso
+      };
+      indicators.forEach(ind => {
+        if (ind.locations[iso]) {
+          row[ind.label] = ind.locations[iso].value;
+        }
+      });
+      return row;
+    });
+  }
+);
+
+const addIndicatorColumn = createSelector(
+  [tableGetSelectedData, getMapIndicator, getSelectedIndicatorHeader],
+  (data, selectedIndicator, selectedIndicatorHeader) => {
```

Comment:
Same here, please add guards for `selectedIndicator` `selectedIndicatorHeader` ;)